subprojects/core/src/integTest/groovy/org/gradle/normalization/ConfigureRuntimeClasspathNormalizationIntegrationTest.groovy

```diff
@@ -350,6 +352,35 @@ class ConfigureRuntimeClasspathNormalizationIntegrationTest extends AbstractInte
         executedAndNotSkipped(project.customTask)
     }
 
+    @IgnoreIf({ AvailableJavaHomes.differentJdk == null })
+    def "property ordering is consistent"() {
```

Comment:
I wasn't able to reproduce the problem by this test. Maybe this only happens between different operating systems?

Reply:
Yeah, I suspect different OS is the only way to make this fail reliably.  Whenever I've seen an ordering issue it is virtually always between different OS.